vertx-auth-mtls/src/main/java/io/vertx/ext/auth/mtls/impl/SpiffeIdentityValidator.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.vertx.ext.auth.mtls.impl;
+
+import io.vertx.ext.auth.mtls.CertificateIdentityValidator;
+
+/**
+ * {@link CertificateIdentityValidator} implementation for SPIFFE certificates for validating SPIFFE identities.
+ * SPIFFE is a URI, present as part of SAN of client certificates, it uniquely identifies client.
+ */
+public class SpiffeIdentityValidator implements CertificateIdentityValidator
+{
+    private static final String SPIFFE_ID_PREFIX = "spiffe://";
```

Comment:
The same constant is define in multiple classes. Another sign that those classes should be a single one.